server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/AuthenticationContract.scala

```diff
@@ -51,41 +50,41 @@ trait AuthenticationContract {
   class BothAuthenticationMechanisms {
     @Tag(CategoryTags.BASIC_FEATURE)
     @Test
-    def shouldRespond200WhenBasicAuthValidAndJWTInvalid(): Unit = {
+    def shouldRespond401WhenBasicAuthValidAndJWTInvalid(): Unit = {
       `given`
         .headers(getHeadersWith(BOB_BASIC_AUTH_HEADER))
         .header(new Header(AUTHORIZATION_HEADER, s"Bearer $UNKNOWN_USER_TOKEN"))
         .body(ECHO_REQUEST_OBJECT)
       .when
         .post
       .`then`
-        .statusCode(SC_OK)
+        .statusCode(SC_UNAUTHORIZED)
     }
 
     @Tag(CategoryTags.BASIC_FEATURE)
     @Test
-    def shouldRespond200WhenJWTAuthValidAndBasicAuthInvalid(): Unit = {
+    def shouldRespond401WhenJWTAuthValidAndBasicAuthInvalid(): Unit = {
       `given`
         .headers(getHeadersWith(new Header(AUTHORIZATION_HEADER, s"Basic ${toBase64(s"this-thing-wrong")}")))
         .header(new Header(AUTHORIZATION_HEADER, s"Bearer $USER_TOKEN"))
         .body(ECHO_REQUEST_OBJECT)
       .when
         .post
       .`then`
-        .statusCode(SC_OK)
+        .statusCode(SC_UNAUTHORIZED)
     }
 
     @Tag(CategoryTags.BASIC_FEATURE)
     @Test
-    def shouldRespond200WhenBothAuthenticationValid(): Unit = {
+    def shouldRespond401WhenBothAuthenticationValid(): Unit = {
```

Comment:
I think we decided to reject this case. I think it's better because supposing you have 2 auth system working (and you are not aware of it), once one break, both will break, while you still have one that is valid.